src/config.rs

```diff
@@ -710,6 +725,10 @@ pub struct CommandLineArgs {
     #[arg(short = 't', long = "tmux")]
     run_in_tmux: bool,
 
+    /// The preferred way to run the new tmux session
+    #[arg(long = "tmux-session-attach-mode")]
+    tmux_session_attach_mode: Option<TmuxSessionAttachMode>,
```

Comment:
Is it necessary to add this CLI option, I think this feature is more like a static feature, one won't need to frequently change it 🤔

Reply:
removed